default.json

```diff
@@ -27,6 +27,11 @@
       "schedule": [
         "after 2am and before 3am"
       ]
+    },
+    {
+      "matchDatasources": ["github-tags"],
```

Comment:
I can find in the docs if `"github-tags"` is the default or not.

Reply:
/cc @danielroe might know some more context why it is needed